qodana.yaml

```diff
@@ -0,0 +1,29 @@
+#-------------------------------------------------------------------------------#
+#               Qodana analysis is configured by qodana.yaml file               #
+#             https://www.jetbrains.com/help/qodana/qodana-yaml.html            #
+#-------------------------------------------------------------------------------#
+version: "1.0"
```

Comment:
The version is set to "1.0", which is fine if it matches the version of Qodana you are using. Ensure this is correct.